id/community/ruby-core/index.md

```diff
@@ -12,65 +12,40 @@ Sehingga, dari mana Anda memulai?
 
 Topik seputar pengembangan Ruby yang dibahas disini adalah:
 
-* [Menggunakan Subversion untuk Memonitor Pengembangan Ruby](#following-ruby)
-* [Bagaimana Menggunakan Git pada Repository Utama Ruby](#git-ruby)
+* [Menggunakan Git untuk Memonitor Pengembangan Ruby](#following-ruby)
 * [Menyempurnakan Ruby, Patch by Patch](#patching-ruby)
-* [Peraturan untuk Developer Inti](#coding-standards)
+* [Peraturan untuk Core Developer](#coding-standards)
```

Comment:
Why do we change this?

Reply:
I don't know. I feel "Core Developer" is better than "Developer Inti". What do you think @gozali?

Well, "Developer Inti" also makes sense.

Reply:
Hi @gozali, any update on this?

Reply:
OK cool